.docs/index.md

```diff
@@ -3,50 +3,24 @@
 Openly serious about storage
 
 ---
-REX-Ray delivers persistent storage access for container runtimes, such as
-Docker and Mesos, and provides an easy interface for enabling advanced storage
-functionality across common storage, virtualization and cloud platforms. For
-example, here's how to list the volumes for a VirtualBox VM running Linux with
-REX-Ray:
+REX-Ray is an open source storage volume manager for container runtimes, such as Docker and Mesos. REX-Ray enables stateful applications, such as databases, key/value stores or any containerized app, to persist and maintain it's data after the life cycle of the container has ended. Built-in High Availability allows orchestrators such as [Docker Swarm](https://docs.docker.com/engine/swarm/), [Kubernetes](http://kubernetes.io/), and [Mesos Frameworks](http://mesos.apache.org/) like [Marathon](https://mesosphere.github.io/marathon/) to automatically orchestrate storage tasks between hosts in a cluster.
 
-```sh
-$ rexray volume --service virtualbox
-- attachments:
-  - instanceID:
-      id: e71578b0-1bfb-4fa5-bcd5-4ae982fd4a9b
-      driver: virtualbox
-    status: /Users/akutz/VirtualBox/libStorage/libStorage.vmdk
-    volumeID: 1b819454-a280-4cff-aff5-141f4e8fd154
-  name: libStorage.vmdk
-  size: 64
-  status: /Users/akutz/VirtualBox/libStorage/libStorage.vmdk
-  id: 1b819454-a280-4cff-aff5-141f4e8fd154
-  type: ""
-```
+Built on top of the [libStorage](http://libstorage.readthedocs.io/en/stable) framework, REX-Ray's simplified architecture consists of a single binary and runs as a stateless service on every host using a configuration file to orchestrate multiple storage platforms.
+
+![REX-Ray Architecture](http://i.imgur.com/LR9Hvig.gif)
 
-## Overview
-REX-Ray is a storage orchestration tool that provides a set of common commands
-for managing multiple storage platforms. Built on top of the
-[libStorage](http://libstorage.readthedocs.io/en/stable) framework, REX-Ray
-enables persistent storage for container runtimes such as Docker and
-Mesos.
+!!! note "note" 
 
-!!! note "note"
+    The current REX-Ray release omits support for several, previously verified storage platforms. These providers will be reintroduced incrementally, beginning with 0.4.1. If an absent driver prevents the use of REX-Ray, please continue to use 0.3.3 until such time the storage platform is re-introduced as a part of the [libStorage](http://libstorage.readthedocs.io/en/stable) framework. Instructions on how to [install](./user-guide/installation.md#rex-ray-033) and [configure](http://rexray.readthedocs.io/en/v0.3.3) REX-Ray 0.3.3 are both available.
```

Comment:
Hi @kacole2,

I've previously noted for sake of readability and long-term support it's a documented request to limit Markdown to a width of 80 columns. Please reformat the above text. Thanks!